api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/audit/ClaimAuditor.java

```diff
@@ -27,10 +30,18 @@ public void auditNewClaim(Claim claim) {
             return;
         }
         NewClaimEvent newClaimEvent = NewClaimEvent.builder()
-                .claimantId(claim.getId())
+                .claimId(claim.getId())
                 .claimStatus(claim.getClaimStatus())
                 .eligibilityStatus(claim.getEligibilityStatus())
                 .build();
         eventLogger.logEvent(newClaimEvent);
     }
+
+    public void auditNewCard(UUID claimId, CardResponse cardResponse) {
+        NewCardEvent newCardEvent = NewCardEvent.builder()
+                .claimId(claimId)
+                .cardAccountId(cardResponse.getCardAccountId())
```

Comment:
We should check for a null `CardResponse` as the new claim event logging does above.

Reply:
Done.